CMSgov/mint-app
# Easi 2366 Collaborator Access Wrapper

# EASI-2366

## Changes and Description

- Added a wrapper to ensure non-collaborators cannot access any editable part of a model plan
- Cypress testing

MINT_ASSESSMENT role granted access regardless of collaborator status

## How to test this change

- Seed the database
- Log in as anyone other than MINT
- Attempt to enter any model plan either by clicking table link or manually entering in URL
- Ensure you are always directed to read-only view

- Create a model plan, or log in as MINT
- Enter the/a model plan and ensure you have all the normal collaborator access control

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
